demo_nf_logs.py

```diff
@@ -0,0 +1,86 @@
+import asyncio
```

Comment:
I'm wondering if this would make more sense as a local recipe in `orca-recipes`

Reply:
Yes, let's add it to orca recipes!

Reply:
I have removed the script and `pandas` from the `dev` dependencies of this repo and opened [this](https://github.com/Sage-Bionetworks-Workflows/orca-recipes/pull/28) PR in `orca-recipes` to add the script and bump the `py-orca` version. The `orca-recipes` PR will be finalized after this one has been merged and a new version has been minted.